doc/src/records/dhwheater.md

```diff
@@ -98,6 +120,18 @@ Storage tank volume. Must be omitted or 0 for instantaneous whTypes.  Used by HP
               runtime errors) 
   -----------------------------------------------------------------------
 
+**whVolRunning=*float***
+
+Running storage volume = vol above aquastat. Determines require total volume based on aquastat position. Ecotope's HPWH tank and heater.
```

Comment:
Because this is user facing documentation, we should avoid unnecessary abbreviations like "vol". There are a lot of these in the CNRECS comments, but they should be spelled out here (with exceptions for well known or previously defined acronyms in the same file).